Tasks/AzureRmWebAppDeploymentV4/operations/KuduServiceUtility.ts

```diff
@@ -423,37 +423,55 @@ export class KuduServiceUtility {
         }
     }
 
-    private _getUpdateHistoryRequest(isDeploymentSuccess: boolean, deploymentID?: string, customMessage?: any): any {
+    private _getUpdateHistoryRequest(isDeploymentSuccess: boolean, deploymentID?: string, customMessage?: any, artifactAlias?: string): any {
```

Comment:
P2: given that this is private method and you are always passing artifactAlias why is it nullable?

Reply:
using env variable now for artifact alias